client/src/auth/signin.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import axios from 'axios';
+
+class Signin extends React.Component {
+    state = {
+        username: 'England',
+        password: 'France',
+    };
+
+    render() {
+        return (
+            <form onSubmitHadler={this.submitHandler}>
```

Comment:
This attribute should just be called `onSubmit`. You could also skip the `form` tags and just use `onClick` as a property on the button.